infra.go

```diff
@@ -92,15 +92,15 @@ func (i *InfrastructureResource) GetTags(client *Client, variables *PayloadVaria
 	return &q.Account.InfrastructureResource.Tags, nil
 }
 
-func (i *InfrastructureResource) ResourceId() ID {
-	return *NewID(i.Id)
+func (infrastructureResource *InfrastructureResource) ResourceId() ID {
+	return *NewID(infrastructureResource.Id)
 }
 
-func (i *InfrastructureResource) ResourceType() TaggableResource {
+func (infrastructureResource *InfrastructureResource) ResourceType() TaggableResource {
 	return TaggableResourceInfrastructureresource
 }
 
-func (client *Client) CreateInfrastructure(input InfraInput) (*InfrastructureResource, error) {
+func (client *Client) CreateInfrastructure(input InfraInput) (*InfrastructureResource, []OpsLevelWarnings, error) {
```

Comment:
Adding warnings to the return params will break `createInfraCmd` in the cli and possibly elsewhere. We should discuss what to do with warnings and should more than likely capture these in logs since warnings don't generally effect the flow of a program

Reply:
Reverted this and re-added the `TODO` comment, I am avoiding adding logging since we don't want a library to log